src/sentry/static/sentry/app/components/organizations/headerItemPosition.jsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'react-emotion';
+import {AutoCompleteRoot} from 'app/components/dropdownAutoCompleteMenu';
+import {TimeRangeRoot} from 'app/components/organizations/timeRangeSelector/index.jsx';
+
+const HeaderItemPosition = styled.div`
```

Comment:
shouldn't this all just go in headerItem? I can't think of a situation where we *wouldn't* want this behavior, and it seems weird to add both components every time.

Reply:
the problem is that `HeaderItem` sits at the bottom of the stack... they are at least 2-3 levels deep from where we actually need the rules to make them 1/3 the width of `Header` and to set the max-width. Ideally `HeaderItem` would live at this level instead, so maybe we can explore merging them here.

Reply:
oh right because dropdownAutoComplete wraps it in the actor component and stuff